packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache

```diff
@@ -1,11 +1,8 @@
 <div id="avatar-change">
-  <section class="modal-panel">
+    <section class="modal-panel">
```

Comment:
This indentation change looks wrong? The `section` on line 2 should be in between the indentation level of the `div` elements on lines 1 and 3?

Reply:
Ugh, not sure how changed, maybe code editor? I'll clean this up.